runtime/backtrace_byt.c

```diff
@@ -260,7 +260,10 @@ void caml_stash_backtrace(value exn, value * sp, int reraise)
 code_t caml_next_frame_pointer(value ** sp, value ** trsp)
 {
   while (*sp < Caml_state->stack_high) {
-    code_t *p = (code_t*) (*sp)++;
+    value spv = (*sp)++;
+    code_t *p;
+    if (Is_long(spv)) continue;
```

Comment:
So it should be dereferenced here.
```suggestion
    if (Is_long(*spv)) continue;
```